package.json

```diff
@@ -39,13 +39,13 @@
     "expo": "40.0.0",
     "expo-asset": "8.2.1",
     "expo-constants": "9.3.3",
-    "expo-document-picker": "9.1.2",
-    "expo-file-system": "11.0.2",
+    "expo-document-picker": "~8.4.1",
```

Comment:
Please no carret and no need to change version of the version of any package for wording fix

Reply:
Okay, I will fix this. Thanks for informing.